doc/modbus_write_file_record.txt

```diff
@@ -0,0 +1,37 @@
+modbus_write_file_record(3)
+===========================
+
+
+NAME
+----
+modbus_write_file_record - read registers from record in a file
+
+
+SYNOPSIS
+--------
+*int modbus_write_file_record(modbus_t *'ctx', int 'addr', int 'sub_addr', int 'nb', uint16_t *'src');*
```

Comment:
in the code, it is `const uint16_t *src`.

Reply:
Right, will fix